server/src/main/java/org/opensearch/search/profile/query/CollectorResult.java

```diff
@@ -120,35 +173,75 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeString(collectorName);
         out.writeString(reason);
         out.writeLong(time);
+        out.writeLong(reduceTime);
```

Comment:
At the and and please:

````
        if (out.getVersion().onOrAfter(Version.V_3_0_0)) {
              out.writeLong(reduceTime);
              out.writeLong(maxSliceTime);
              out.writeLong(minSliceTime);
              out.writeLong(avgSliceTime);
              out.writeVInt(sliceCount);
         }
```

Reply:
Hi @reta, thanks for the code review. I have implemented the suggested changes.